src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -226,11 +240,24 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
                     var errorMessage = ExceptionUtilities.DisplayMessage(ex);
                     _logger.Log(ProjectManagement.MessageLevel.Error, errorMessage);
 
-                    _loadingStatusIndicator.SetError(errorMessage);
-
                     _loadingStatusBar.SetError();
                     _loadingStatusBar.Visibility = Visibility.Visible;
                 }
+                finally
+                {
+                    if (_loadingStatusIndicator.Status != LoadingStatus.NoItemsFound)
```

Comment:
Think the entire Finally should be a Helper Method. eg, `void RemoveLoadingIndicatorIfPresent(List<stuff> Items)`

Reply:
We only have this case in this finally block. That would lead to a helper method with only one usage.

Reply:
Following up in another issue. https://github.com/NuGet/Home/issues/9219